frontend/src/pages/signup/Signup.tsx

```diff
@@ -2,19 +2,12 @@ import React, { useState, useContext } from "react";
 import { AuthContext } from "../../contexts/authContext";
 import { Link } from "react-router-dom";
 import { useNavigate } from "react-router-dom";
-import {
-  Box,
-  Grid,
-  Typography,
-  TextField,
-  Button,
-  Checkbox,
-  FormControlLabel,
-  InputLabel,
-} from "@mui/material";
+import { Typography, TextField, Button, InputLabel } from "@mui/material";
 import signupImage from "../../assets/signup.svg";
 import Icon from "../../assets/aperture.svg";
 import "./Signup.css";
+import axios from "axios";
+
```

Comment:
Great Job @hamoudi-hamoudi , just one thing does this work? Should not we use the axios from the API file? https://github.com/HackYourFuture-CPH/class-25-ironside/blob/dev/frontend/src/api/index.ts ? In the future, we will want to use our URL

Reply:
Yes its working :grin: , the URL you mention  http://localhost:3000/ ?

Reply:
I see now, it uses `localhost` as the fallback URL, but once the app is published it will need to use the Backend URL which will replace this variable  `process.env.REACT_APP_API_URL ` In the API file, but if you do not have it it will fall to localhost again, so it will work same as now

Reply:
so we need to import axios from that file, it will look as this `import axios from '../../api'` and the function changes to this `await axios.post("/api/user", newUser, config);` hmm I hope I "explained it well" if its not clear we can talk about this more later